### PR TITLE
Correct format of TimeEntries on Reports. Closes #488, #485

### LINKED ISSUE #488: Errors on Bartchar due to misreading of Report data
Seems to be theres is a lack of understanding between app and Reports API. The problem is present on data that feed the Barchart. 
Could be simulated with test account and year zoom level ( the piechart shows data but the barchart not).

### PR DESCRIPTION
**Pull request by monday8am**
- Correct format.
- Now Barchart labels are formatted with the format selected by the user ( with minor modifications). The final result is equal to web barchart.

Fixes #488, #485

### CODE REVIEW

File: Phoebe/Data/Models/TimeEntryModel.cs [hunk 1]
```diff
@@ -229,7 +229,7 @@ public bool IsBillable
         public string GetFormattedDuration ()
         {
             TimeSpan duration = GetDuration (Data, Time.UtcNow);
-            string formattedString = duration.ToString (@"h\:mm\:ss");
+            string formattedString = duration.ToString (@"hh\:mm\:ss");
```
Comment (roosmaa) [inline Phoebe/Data/Models/TimeEntryModel.cs]: Why this change? Do we want to display duration in "08:00:00" format? Will UI survive the extra 0?

Comment (monday8am) [inline Phoebe/Data/Models/TimeEntryModel.cs]: I searched and your previous uses of this format include the two "hh". 

https://github.com/toggl/mobile/blob/314cbd59544b372c126d544c75487f22c72ab6a7/Ross/ViewControllers/TimerNavigationController.cs
https://github.com/toggl/mobile/blob/e6ab7a7880bd4284ef0c7e120ea7f74a3d45ef28/Joey/UI/Adapters/LogTimeEntriesAdapter.cs

The UI is prepared to accept two digits in this format ( 12 for example).

**Comment by roosmaa** [comment]
Looks OK. Didn't try to reproduce the bugs to see if it actually fixes them - @taavirehemagi can you do that? :angel:

**Comment by taavirehemagi** [comment]
OK from me. Fixes the bugs. :+1:
